### ISSUE TITLE
Add new user triggers long syncing 

### ISSUE THREAD

**Issue by LIL717** [ceaseless-prayer/CeaselessIOS#135]
I'm guessing that the table could be visible for this update because ultimately it is only adding one user so it shouldn't be jittery.   Would be good to try it.

**Comment by chrislim** [comment]
Another way to deal with it is to take the user straight to the newly created person card.

**Comment by chrislim** [comment]
recent push to contacts-view does what I suggested.

**Comment by LIL717** [comment]
that looks good, but when you go back to the person view it is still syncing forever - I did a little testing and as I suspected, there is no problem viewing the table while it is synching when it has already been populated and there is only this one addition.  Propose that we identify when there has only been one addition and do not hide the table view while syncing in that case.  Also at line 714, I believe it is better to push the person view before dismissing the New Person view, otherwise there is a brief view of the People table and then it takes you to the Person View.  With those 2 lines swapped, the Person View is visible on return and pressing the back button takes you to the People table.
